almalinux-deploy.sh

```diff
@@ -25,7 +25,7 @@ BRANDING_PKGS=("centos-backgrounds" "centos-logos" "centos-indexhtml" \
                 "centos-logos-ipa" "centos-logos-httpd" \
                 "oracle-backgrounds" "oracle-logos" "oracle-indexhtml" \
                 "oracle-logos-ipa" "oracle-logos-httpd" \
-                "oracle-epel-release-el8" \
+                "oracle-epel-release-el8" "oracle-epel-release-el9"\
```

Comment:
Please ad a space after the `"oracle-epel-release-el9"`

Reply:
Fixed